### PR TITLE
splits events/<event-id>/tickets/discount-codes into proper pages

### LINKED ISSUE #291: Split events/<event-id>/tickets/discount-codes into proper pages
**I'm submitting a ...**  (check one with "x")
- [ ] bug report
- [x] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/open-event-frontend

`events/<event-id>/tickets/discount-codes` - All codes
`events/<event-id>/tickets/discount-codes/active` - Active codes
`events/<event-id>/tickets/discount-codes/inactive` - inactive codes

### PR DESCRIPTION
**Pull request by anu-007**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
splits events/<event-id>/tickets/discount-codes into proper pages

#### Changes proposed in this pull request:
splits events/<event-id>/tickets/discount-codes into-

-`events/<event-id>/tickets/discount-codes`
-`events/<event-id>/tickets/discount-codes/active`
-`events/<event-id>/tickets/discount-codes/inactive`

demo- https://open-event-frontend-branch3.herokuapp.com/events/963d3ba5/tickets/discount-codes

Fixes #291

### CODE REVIEW

File: app/templates/events/view/tickets/discount-codes.hbs [hunk 1]
```diff
@@ -1,93 +1,24 @@
-<div class="ui stackable grid">
+<div class="ui grid stackable">
+  <div class="row">
+    <h2 class="ui header eight wide column left floated {{if device.isMobile 'center' 'left'}} aligned">{{t 'Discount Codes'}}</h2>
+    <div class="eight wide column {{if device.isMobile 'center' 'right'}} aligned"><button class="ui button blue"><i class="plus icon"></i>{{t 'Create a discount'}}</button></div>
```
Comment (CosmicCoder96) [inline app/templates/events/view/tickets/discount-codes.hbs]: @anu-007 Everything else looks fine, can you please split this into two lines, for better readability. It's too long.

Comment (niranjan94) [inline app/templates/events/view/tickets/discount-codes.hbs]: Do it for both h2 and the div.

**Comment by anu-007** [comment]
@CosmicCoder96 @niranjan94 please review.

File: app/templates/events/view/tickets/discount-codes/list.hbs [hunk 2]
```diff
@@ -0,0 +1,66 @@
+<div class="sixteen wide column">
+  <table class="ui stackable very basic table">
```
Comment (geekyd) [inline app/templates/events/view/tickets/discount-codes/list.hbs]: Add `compact` class here.

Comment (anu-007) [inline app/templates/events/view/tickets/discount-codes/list.hbs]: added `compact` class please review.thanks!!

**Comment by harshitagupta30** [comment]
Hi @anu-007! Please resolve the conflicts.

**Pull request merged**